cpp/src/cylon/util/arrow_utils.cpp

```diff
@@ -193,5 +193,36 @@ arrow::Status free_table(const std::shared_ptr<arrow::Table> &table) {
   return arrow::Status::OK();
 }
 
+arrow::Status duplicate(const std::shared_ptr<arrow::ChunkedArray>& cArr,
```

Comment:
should we make this uppercase?

Reply:
Lets keep these internal methods lowercase